kitchen.yml

```
@@ -429,64 +429,17 @@ verifier:
 
 suites:
   - name: default
-    excludes:
-      - gentoo-stage3-latest-master-py3
-      - gentoo-stage3-systemd-master-py3
-      - gentoo-stage3-latest-3003-1-py3
-      - gentoo-stage3-systemd-3003-1-py3
-      - gentoo-stage3-latest-3002-6-py3
-      - gentoo-stage3-systemd-3002-6-py3
-      - gentoo-stage3-latest-3001-7-py3
-      - gentoo-stage3-systemd-3001-7-py3
-      - gentoo-stage3-latest-3000-9-py3
-      - gentoo-stage3-systemd-3000-9-py3
     provisioner:
       state_top:
         base:
           '*':
             - TEMPLATE._mapdata
             - TEMPLATE
-      pillars:
-        top.sls:
-          base:
-            '*':
-              - TEMPLATE
-              - define_roles
+      pillars: {}
       pillars_from_files:
         TEMPLATE.sls: pillar.example
-        define_roles.sls: test/salt/pillar/define_roles.sls
-    verifier:
-      inspec_tests:
-        - path: test/integration/default
-  - name: gentoo
-    includes:
-      - gentoo-stage3-latest-master-py3
-      - gentoo-stage3-systemd-master-py3
-      - gentoo-stage3-latest-3003-1-py3
-      - gentoo-stage3-systemd-3003-1-py3
-      - gentoo-stage3-latest-3002-6-py3
-      - gentoo-stage3-systemd-3002-6-py3
-      - gentoo-stage3-latest-3001-7-py3
-      - gentoo-stage3-systemd-3001-7-py3
-      - gentoo-stage3-latest-3000-9-py3
-      - gentoo-stage3-systemd-3000-9-py3
-    provisioner:
-      state_top:
-        base:
-          '*':
-            - TEMPLATE._mapdata
-            - TEMPLATE
-      pillars:
-        top.sls:
-          base:
-            '*':
-              - TEMPLATE
-              - gentoo
-              - define_roles
-      pillars_from_files:
-        TEMPLATE.sls: pillar.example
-        gentoo.sls: test/salt/pillar/gentoo.sls
-        define_roles.sls: test/salt/pillar/define_roles.sls
+      pillars_from_directories:
+        - test/salt/pillar
```

Will this still find `pillar.example` for the `TEMPLATE` pillar if it only looks in `test/salt/pillar`?

Or will we need to use `pillars_from_files` alongside `pillars_from_directories`?

```suggestion
      pillars_from_files:
        TEMPLATE.sls: pillar.example
      pillars_from_directories:
        - test/salt/pillar
```

---

You are completely right, in my PR, I [kept](https://github.com/saltstack-formulas/template-formula/blob/7fc998282d9ee679db6a6ccb1cb76b2bf79938eb/kitchen.yml#L438) the `TEMPLATE.sls: pillar.example` in `pillars_from_files`

---

@baby-gnu Ah, the diff caught me out, with the two suites being merged into one.